python-bioformats/meta.yaml

```diff
@@ -30,15 +30,21 @@ requirements:
   build:
     - python
     - setuptools
-    - javabridge >=1.0
+    - boto3 >=1.14.23
```

Comment:
surely we need `openjdk` as a build dep too (to ensure that we build and find the right java)?

Reply:
actually, we can probably drop openjdk from both build and run dependencies as it should be pulled in by the python-javabridge package.

Reply:
It is needed as a build for`python-javabridge`, but there's no java involved in the `python-bioformats` build. It is needed in run, but you're right, `python-javabridge` should take care of it.